docs/running-on-yarn.md

```diff
@@ -419,6 +419,15 @@ To use a custom metrics.properties for the application master and executors, upd
   in YARN ApplicationReports, which can be used for filtering when querying YARN apps.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.app.priority</code></td>
```

Comment:
We don't have any configs that use `.app`, so I'd call this just `spark.yarn.priority`.